### ISSUE TITLE
Adding ovlytics 

### ISSUE THREAD

**Issue by ethanellison** [openvolley/pydatavolley#4]
The python implementation looks great and I cant wait to start using it! Any plans to develop a python version the ovlytics R package as well? Particularly the augment_plays function. I use it specifically for the first contact quality column. Thanks!

**Comment by tyler-widdison** [comment]
Ethan, great idea. I'll add those columns in as the normal read so they appear.

**Comment by napo** [comment]
Hi @ethanellison 
This is my proposal:
let's create a module called "analysis" inside the pydatavolley package in order to use it in this way
```python
from pydatavolley import read_dv
from pydatavolley import analysis
```
from here we are going to create the various methods which are not other than those of the package https://ovlytics.openvolley.org/reference/index.html
Eg.
```python
# read data
dvf = read_dv.DataVolley(datavolleyfile)
# extract all the actions
plays = dvf.get_plays()
attacks_efficiency = analysis.attack_eff(plays)
```
if @tyler-widdison  agrees I proceed with the development and make a pull request

**Comment by tyler-widdison** [comment]
@napo I think that is a great idea. Make the pull request!

**Comment by raymondben** [comment]
Minor suggestion: to make it easier for users who might be switching between the R and Python versions, it might be worth keeping naming consistent where it's reasonable to do so. I am thinking of module (package) names as well as functions. So, e.g. instead of an "analysis" module, call it "ovlytics" to make it clear that it's filling the same role as the R package of that name?
(The downside is that the R package names might not always be ideal, partly because they need to be unique package names in their own right. An R package named "analysis" would be pretty odd, a lot of R packages do analysis, you wouldn't know without reading the docs that it was even volleyball-related. But for the Python modules it is clear that the "analysis" module lives within the pydatavolley package. So maybe go for consistent Python/R naming where it's sensible.)

**Comment by ethanellison** [comment]
@napo i agree with the approach for the analytical methods but the augment plays method i think should be built in to the read_dv as optional columns to be included as output as per @tyler-widdison initial suggestion.

**Comment by napo** [comment]
@ethanellison your point of view is clear to me, but I believe that it remains useful to allow users to change some weights via variables.
I think we can solve with this : I'll add a variable (eg. calculate_efficiency=true) to the read_dv method with which to calculate the analyzes with the predefined weights.
What do you think?

**Comment by napo** [comment]
@raymondben  I completely agree with your point of view.
Before writing the comment I was also very undecided whether to propose ovlytics or analysis. 
As you yourself write, using analisys as a pydatavolley module is very clear.
The doubt that comes to me is if one day we are going to develop pyperanavolley (the possible Python library to read the peranavolley format) whether we should create the ovlytics module inside the package, or if instead it is better to create the parser of the peranavolley format inside pydatavolley.
If it's ok with you I'll start writing the ovlytics module inside pydatavolley using the name ovlytics.
Otherwise we propose it as an external module in a new repository with the name pyovlytics but it doesn't seem very advantageous to me.

**Comment by napo** [comment]
@raymondben now there is a first implementation of 
[ovlytics](https://github.com/openvolley/pydatavolley/blob/main/datavolley/ovlytics.py)

you can use in the code so
```python
from datavolley import ovlytics as statistics
```
